README.Rmd

```diff
@@ -400,17 +400,21 @@ To use sparklyr with Databricks Connect first launch a Cluster on Databricks. Th
     * Port (default port number is `15001`)
 
 To configure `sparklyr` with Databricks Connect, set the following environment variables:
-```{bash}
+```{bash, eval=FALSE}
 export SPARK_VERSION=2.4.4
 export SPARK_HOME=`databricks-connect get-spark-home`
 ```
 
 Now simply create a spark connection as follows
-```{r eval=FALSE}
-sc <- spark_connect(method = "databricks")
+```{r, eval=FALSE}
+spark_home <- system("databricks-connect get-spark-home")
+sc <- spark_connect(method = "databricks",
+                    spark_home = spark_home)
```

Comment:
Do you need to set `spark_home` if `SPARK_HOME` is already set?

Reply:
Not really -- but this was what's shown on the  screenshot provided so I figured the code here should match the screenshot